js/components/PortfolioTable.jsx

```diff
@@ -39,6 +52,7 @@ var PortfolioTable = React.createClass({
         totalDividends += category.investments[j].dividend;
       }
     }
+    totalDividends = Math.floor(totalDividends * 100)/100;
```

Comment:
This fixes a strange error where sometimes adding a sum of numbers together created a really long number.